code/modules/reqs/supplypacks.dm

```diff
@@ -187,33 +187,30 @@ WEAPONS
 		/obj/item/cell/lasgun/tesla,
 		/obj/item/cell/lasgun/tesla,
 	)
-	cost = 60
+	cost = 30
 
 /datum/supply_packs/weapons/recoillesskit
 	name = "Recoilless rifle kit"
 	contains = list(/obj/item/storage/box/recoilless_system)
 	cost = 40
 
-
-/datum/supply_packs/weapons/specgrenadier
-	name = "Grenadier Specialist kit"
-	contains = list(/obj/item/weapon/gun/launcher/m92)
-	cost = 80
-
-/datum/supply_packs/weapons/specscoutm4ra
-	name = "Scout Specialist kit (M4RA)"
-	contains = list(/obj/item/weapon/gun/rifle/m4ra)
-	cost = 80
-
-/datum/supply_packs/weapons/specscoutmbx
-	name = "Scout Specialist kit (MBR-900)"
-	contains = list(/obj/item/weapon/gun/shotgun/pump/lever/mbx900)
-	cost = 80
+/datum/supply_packs/weapons/antimatrifle
+	name = "Anti Material Rifle kit"
+	contains = list(
+		/obj/item/weapon/gun/rifle/sniper/antimaterial,
+		/obj/item/ammo_magazine/sniper,
```

Comment:
Remove magazines from weapon orders in line with all other orderable guns

Reply:
done